packages/shared/styles/components/SfHeader.scss

```
@@ -1,52 +1,83 @@
-@import '../variables';
+@import "../variables";
 
 $header-mobile-background: $c-light !default;
 $header-mobile-height: 50px !default;
-$header-mobile-justify-content: space-between !default;
+$header-desktop-height: 80px !default;
+
+$header-navigation-padding: 40px !default;
+$header-navigation-justify-content: center !default;
+$header-navigation-list-font-size: $font-size-regular-mobile !default;
+$header-navigation-list-font-weight: 500 !default;
+$header-navigation-list-color: $c-dark !default;
+
+$header-icon-margin: 30px !default;
+$header-icon-cursor: pointer !default;
+
 $header-mobile-align-items: center !default;
+
 $header-mobile-padding-x: 1rem !default;
-$header-tablet-justify-content: flex-start !default;
-$header__logo-width: 21px !default;
+
+$header-mobile-logo-width: 21px !default;
+$header-desktop-logo-width: 34px !default;
+
 $header__logo-height: auto !default;
 
 .sf-header {
   display: flex;
+  align-items: center;
+  width: 100%;
+  margin: {
+    left: auto;
+    right: auto;
+  }
   @media (max-width: $mobile-max) {
```

```suggestion
  @media (min-width: $desktop-min) {
```
we prefere mobile first :)

---

than this styles should be outside `@media`